.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,76 @@
+name: "🐛 Bug Report"
+description: Create a new ticket for a bug.
+title: "🐛 [BUG] - <title>"
+labels: [
+  "bug"
+]
+projects: [
+  "jondavid-black/AaC Prototype"
+]
+body:
+  - type: checkboxes
+    id: pre-requistes
+    attributes:
+      label: "Pre-requisites"
+      description: Please answer the following questions for yourself before submitting an issue.
+      options:
+      - label: I am running the latest version.
+        required: true
```

Comment:
This may be better captured as a text field for the AaC version as users may believe they are running the most up-to date version, but are actually running an older version.

Reply:
Added an input field for getting the aac version being used.